tensorflow/training/docker/2.0.2/py3/Dockerfile.cpu

```diff
@@ -133,7 +133,7 @@ RUN ${PIP} install --no-cache-dir -U \
     "sagemaker<2" \
     sagemaker-experiments==0.* \
     "sagemaker-tensorflow>=2.0,<2.1" \
-    smdebug==0.8.1 \
+    smdebug==0.9.2 \
```

Comment:
Please make this an ARG like the other dockerfiles

Reply:
Done